backend_python/backend/routes.py

```diff
@@ -23,7 +24,7 @@ def endpoint_create_town():
     town = find_town(request.json['town']['slug'])
 
     if not town:
-        town = Town(request.json['town']['slug'])
+        town = Game(request.json['town']['slug'])
```

Comment:
i think it will be confusing to have instance of `Game` named `town`, given we already have `Town` class. i would rename it for clarity